vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
By default those files are in `.vscode` directory, not in `vscode` like in this PR

Reply:
what's more, those files differ depending on user setup. i dont think we should include those in this repo like that. maybe add it to developer guide? I keep mine in my separate repo: https://github.com/dkalinowski/ovms-vscode
I doubt you would want to keep my settings in OVMS repo

Reply:
This was added for windows dev convenience. The READE file instructs how to use it on windows machines. I can move it to the guide not to add it to repository.